src/code/client.coffee

```diff
@@ -547,6 +547,25 @@ class CloudFileManagerClient
     # Check browser support for document.location.origin (& window.location.origin)
     "#{document.location.origin}#{document.location.pathname}#{suffix}"
 
+  # Takes an array of strings representing url parameters to be removed from the URL.
+  # Removes the specified parameters from the URL and then uses the history API's
+  # pushState() method to update the URL without reloading the page.
+  # Adapted from http://stackoverflow.com/a/11654436.
+  removeQueryParams: (params) ->
```

Comment:
It looks like you're supposed to be able to pass this function an array of query params to eliminate. However, as far as I can tell, it will only ever remove the last element of that array.

`hash` is only calculated once, at the start of the function, and `replace` is called on `hash[0]`, which isn't changing.

A quick fix might be on line 562: `url = hash[0] = hash[0].replace(...`, but there may be a more elegant fix.